.github/dependabot.yml

```diff
@@ -11,3 +11,10 @@ updates:
       interval: "weekly"
     reviewers:
       - "FortnoxAB/PlatMap"
```

Comment:
maybe we should use the same casing used when tagging the group in a conversation` @FortnoxAB/platmap` @FortnoxAB/platmap